src/hooks/use-fetch-wpcom-sites.tsx

```diff
@@ -49,7 +49,7 @@ function getSyncSupport( site: SitesEndpointSite, connectedSiteIds: number[] ):
 	if ( connectedSiteIds.some( ( id ) => id === site.ID ) ) {
 		return 'already-connected';
 	}
-	if ( ! site.plan.features.active.includes( STUDIO_SYNC_FEATURE_NAME ) ) {
+	if ( ! site.plan || ! site.plan.features.active.includes( STUDIO_SYNC_FEATURE_NAME ) ) {
```

Comment:
I wonder how reliable are the attributes inside the plan.

```suggestion
	if ( ! site.plan?.features?.active?.includes( STUDIO_SYNC_FEATURE_NAME ) ) {
```

Reply:
I propose to add extra optional chaining points as soon as we face them, since such option looks like overkill, which can just confuse as in the future, to add unnecessary conditions to other parts of codebase. I advocate to use only `plan` as `undefined` value ATM.